src/scss/_themes.scss

```diff
@@ -1,6 +1,6 @@
 @mixin oTeaserCollectionNumbered() {
 	.o-teaser-collection__items {
-		counter: o-teaser-collection-items;
+		counter-reset: o-teaser-collection-items;
```

Comment:
I believe this was meant to be a counter-reset originally.

This will reset the `o-teaser-collection-items` counter whenever a new o-teaser-collection__items is found in the DOM.